packages/app-store/routing-forms/pages/form-edit/[...appPages].tsx

```diff
@@ -42,7 +44,7 @@ export const FieldTypes = [
     value: "textarea",
   },
   {
-    label: "Select",
+    label: "Single selection",
```

Comment:
<img width="1055" alt="Screenshot 2023-05-17 at 6 13 52 PM" src="https://github.com/calcom/cal.com/assets/1780212/3c0a9d8b-5e8e-4444-9f94-2ca8772a4b7f">
It makes it inconsistent with MultiSelect. We should call that as `Multiple Selection` too